update_snapcraft.py

```diff
@@ -75,13 +75,15 @@ def parse_manual_requirements(path):
 
     with open(manual_requirements_path, encoding="utf-8") as req_file:
         for line in req_file:
-            try:
-                # Requirement cannot handle comments
-                req = Requirement(line.split("#", maxsplit=1)[0])
-                manual_requirements.add(str(req))
-            except InvalidRequirement as err:
-                msg = f"ERROR: cannot parse manual requirements\n{err}"
-                sys.exit(msg)
+            # Requirement cannot handle comments
+            package = line.split("#", maxsplit=1)[0]
```

Comment:
You will need a `strip()` at the end, otherwise 
```
line = "         "
package = line.split("#", maxsplit=1)[0]
if package:
    print("still works")
```

Reply:
I think this is the sign that I should stop hammering at this and go have dinner...